src/loaders/ImageLoader.js

```diff
@@ -39,11 +39,32 @@ Object.assign( ImageLoader.prototype, {
 
 			var loader = new XHRLoader();
 			loader.setPath( this.path );
-			loader.setResponseType( 'blob' );
+			loader.setResponseType( undefined !== this.responseType ? this.responseType : 'blob' );
 			loader.setWithCredentials( this.withCredentials );
-			loader.load( url, function ( blob ) {
-
-				image.src = URL.createObjectURL( blob );
+			loader.load( url, function ( response ) {
+
+				if (loader.responseType == THREE.ResponseType.Blob) {
+				
+					image.src = URL.createObjectURL( response );
+					
+				} else if (loader.responseType == THREE.ResponseType.ArrayBuffer) {
```

Comment:
No idea. I assume it will do nothing, but maybe someone with more experience with that can comment...